spec/services/spree/solidus_six_saferpay/cancel_authorized_payment_spec.rb

```diff
@@ -29,30 +28,36 @@ module SolidusSixSaferpay
           let(:payment) { create(:six_saferpay_payment) }
 
           it 'does not cancel the payment' do
-            expect(gateway).not_to receive(:void)
+            allow(gateway).to receive(:void)
```

Comment:
Why is `expect(gateway).not_to receive(:void)` changed to this (and 35) line? It seems to do the same thing.

Reply:
This line creates a "test spy" that stubs the `:void` method on this object. This is necessary so that we can later test if the method has been called with `expect(gateway).to have_received(:void)`.

This "spy-and-stub" style of writing tests is the preferred way of writing tests for solidus extensions according to their rubocop configuration. You will find a more in-depth explanation here: https://thoughtbot.com/blog/a-closer-look-at-test-spies